src/http/actions/actions.dependents.ts

```diff
@@ -159,4 +161,67 @@ export module DependentsActions {
       }
     });
   }
+
+  /**
+   * ! Insert new dependent
+   * * DanBaDo - 2021/12/29
+   * @param name {string}
+   * @param phone {string}
+   * @param direction {string}
+   * @param callback {Function}
+   */
+  export const newDependent = (
+    name: string,
+    phone: string,
+    direction: string,
+    callback: Function
+  ) => {
```

Comment:
This action and the next one can replace the previous ```insertNewDependent``` action in a simplest and reusable way.